frontend/src/app/[locale]/opportunity/[id]/page.tsx

```diff
@@ -20,7 +22,12 @@ import OpportunityIntro from "src/components/opportunity/OpportunityIntro";
 import OpportunityLink from "src/components/opportunity/OpportunityLink";
 import OpportunityStatusWidget from "src/components/opportunity/OpportunityStatusWidget";
 
+type OpportunityListingProps = {
+  params: { id: string };
+} & WithFeatureFlagProps;
+
 export const revalidate = 600; // invalidate ten minutes
+export const dynamic = "force-dynamic";
```

Comment:
I honestly don't understand why this needs to be here since we are already including the generateStaticParams, but without the the production build fails to render the page due to the dependence on searchParams in the withFeatureFlag code